.mergify.yml

```diff
@@ -136,6 +136,40 @@ pull_request_rules:
       dismiss_reviews: {}
       delete_head_branch: {}
 
+  # release-1.8 branch
+  - name: automerge backport release-1.8
+    conditions:
+      - author=mergify[bot]
+      - base=release-1.8
+      - label!=do-not-merge
+      - 'status-success=DCO'
+      - 'check-success=canary'
+      - 'check-success=unittests'
+      - 'check-success=golangci-lint'
+      - 'check-success=codegen'
+      - 'check-success=lint'
+      - 'check-success=modcheck'
+      - 'check-success=pvc'
+      - 'check-success=pvc-db'
+      - 'check-success=pvc-db-wal'
+      - 'check-success=encryption-pvc'
+      - 'check-success=encryption-pvc-db'
+      - 'check-success=encryption-pvc-db-wal'
+      - 'check-success=encryption-pvc-kms-vault-token-auth'
```

Comment:
We have more tests now:

* encryption-pvc-kms-vault-k8s-auth
* lvm-pvc
* multi-cluster-mirroring
* rgw-multisite-testing